pkg/controller/storagecluster/reconcile.go

```diff
@@ -1037,3 +1038,96 @@ func getMonCount(nodeCount int) int {
 
 	return count
 }
+
+func newCleanupJob(sc *ocsv1.StorageCluster, RookImage string, OsdID int, reqLogger logr.Logger) *batchv1.Job {
+	labels := map[string]string{
+		"app": "ceph-toolbox-job",
+	}
+
+	job := &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rook-ceph-toolbox-job-" + string(OsdID),
+			Namespace: sc.Namespace,
+			Labels:    labels,
+		},
+		Spec: batchv1.JobSpec{
+			Template: corev1.PodTemplateSpec{
+				Spec: corev1.PodSpec{
+					Volumes: []corev1.Volume{
+						corev1.Volume{
+							Name: "mon-endpoint-volume",
+							VolumeSource: corev1.VolumeSource{
+								ConfigMap: &corev1.ConfigMapVolumeSource{
+									LocalObjectReference: corev1.LocalObjectReference{
+										Name: "mon-endpoint-volume",
+									},
+									Items: []corev1.KeyToPath{
+										corev1.KeyToPath{
+											Key:  "data",
+											Path: "mon-endpoint",
+										},
+									},
+								},
+							},
+						},
+						corev1.Volume{
+							Name:         "ceph-config",
+							VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}},
+						},
+					},
+					RestartPolicy: corev1.RestartPolicyNever,
+					Containers: []corev1.Container{
+						corev1.Container{
+							Name:  "script",
+							Image: "ROOK_DOWNSTREAM_IMAGE",
+							VolumeMounts: []corev1.VolumeMount{
+								corev1.VolumeMount{
+									MountPath: "/etc/ceph",
+									Name:      "ceph-config",
+									ReadOnly:  true,
+								},
+							},
+							Command: []string{"/bin/bash", "-c"},
+							Args: []string{
+								"ceph osd out osd.FAILED_OSD_ID",   //this command marks osd out
```

Comment:
Does it work with the two commands as separate arguments in the string slice? In my testing the commands were a single multi-line parameter as seen [here](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/toolbox-job.yaml#L38-L43), but maybe this way works fine too.

Reply:
I think we changed it to string slice.